Added test case regarding failure of series/limit unless the expression is simplified

Shardul555 opened the pull request:
#### References to other Issues or PRs
#9173


#### Brief description of what is fixed or changed

Initially the issue was the failure of series/limit unless the expression is simplified first, but it is resoved in current master.
So I have added test case addressing this one.


#### Release Notes
<!-- BEGIN RELEASE NOTES -->

NO ENTRY

<!-- END RELEASE NOTES -->

Shardul555 committed:
Added test case addressing issue #9173

Initially the issue was the failure of series/limit unless the expression is simplified first, but it is resoved in current master.
So I have added test case addressing this one.

Shardul555 commented:
@czgdp1807 Can you guide me what should I do so that the required checks will not fail

suryam35 commented:
@Shardul555 your file contains trailing whitespaces 
Refer here [https://github.com/sympy/sympy/wiki/Development-workflow](url) to understand the workflow process

Shardul555 committed:
Added test case regarding failure of series/limit unless the expression is simplified

Added the test case regarding failure of series/limit unless the expression is simplified. It is improvement on last commit as last one had unnecessary whitespaces.

sidhu1012 commented on sympy/series/tests/test_series.py:
```diff
@@ -202,6 +202,13 @@ def test_issue_9549():
 
 
+def test_issue_9173():
+    var('p_0 p_1 p_2 p_3 b_0 b_1 b_2')
```
You are not storing them in variables which you are using, that's why test is failing.

Shardul555 replied:
Thanks for guidance @sidhu1012

Shardul555 committed:
Added test case regarding failure of series until expression is simplified

This pull request will add the test case regarding failure of series problem until th the expression is simplified using .simplify() function.

sidhu1012 commented on sympy/series/tests/test_series.py:
```diff
@@ -202,6 +202,14 @@ def test_issue_9549():
 
 
+def test_issue_9173():
+    from sympy import var
```
Imports should go on top of file.

Shardul555 replied:
Ok I will do it after these checks will be completed.

jksuom replied:
And `var` should not be used in library code.

Shardul555 replied:
Ok thank you @jksuom ,so can you pease suggest me what to use instead of var or simply where I will find the rules related to uses of variables.

0sidharth replied:
As used in other tests like `test_issue_12791`, you can use `symbols` instead

0sidharth replied:
As used in other tests like `test_issue_12791`, you can use `symbols` instead

Shardul555 commented:
Can anyone check on their installed version of sympy output of this series for which the test case is added, I am getting different answers on sympy live shell and on my installed version which is pointing towards git repository of sympy.

Shardul555 committed:
Added test case regarding failure of series until expression is simplified
